sympy/core/power.py

```diff
@@ -1027,6 +1027,8 @@ def _eval_is_algebraic(self):
         if self.base.is_zero or (self.base - 1).is_zero:
             return True
         elif self.exp.is_rational:
+            if not self.base.is_algebraic:
```

Comment:
`if self.base.is_algebraic is False` (it could be None, which would trigger the code here)